config/config.go

```diff
@@ -34,6 +34,7 @@ type License struct {
 	FeaturesToInclude   string `yaml:"features_to_include,omitempty"`
 	MonitorUsers        bool   `yaml:"monitor_users"`
 	MonitorReservations bool   `yaml:"monitor_reservations"`
+	MonitorVersions     bool   `yaml:"monitor_versions"`
```

Comment:
Hey @mjtrangoni ,
thanks for your answer!
I rebased against master and also added "omitempty" where you asked.
I did not explicitly add "False" as a default value, because go and "omitempty" do this as a default. 
So I tested it without the MonitorVersions value and the exporter does automaticly what you wanted!

I squashed the changes into the PR.